packages/block-library/src/query/utils.js

```diff
@@ -375,7 +379,14 @@ export const useUnsupportedBlocks = ( clientId ) => {
 			getClientIdsOfDescendants( clientId ).forEach(
 				( descendantClientId ) => {
 					const blockName = getBlockName( descendantClientId );
-					if ( ! blockName.startsWith( 'core/' ) ) {
+					const blockInteractivity = getBlockSupport(
+						blockName,
+						'interactivity'
+					);
```

Comment:
We can include as well non interactive blocks that supports client navigation (disabling force page reload). Those ones should have this block supports structure. You can check the paragraph block for example.

```
	"interactivity": {
			"clientNavigation": true
		}
```